### PR TITLE
Upgrade to flutter 2

### PR DESCRIPTION
**Pull request by nbrownus**
This PR currently does not address any new functionality we could/should be using, just gets the build working.

### CODE REVIEW

File: lib/components/config/ConfigButtonItem.dart [hunk 1]
```diff
@@ -21,21 +20,5 @@ class ConfigButtonItem extends StatelessWidget {
           constraints: BoxConstraints(minHeight: Utils.minInteractiveSize, minWidth: double.infinity),
           child: Center(child: content),
         ));
-
```
Comment (JohnMaguire) [inline lib/components/config/ConfigButtonItem.dart:24]: This change seems kind of major compared to the other API fixes in this PR. Can you explain why we are removing this?

File: lib/screens/siteConfig/CertificateScreen.dart [hunk 2]
```diff
@@ -219,34 +219,22 @@ class _CertificateScreenState extends State<CertificateScreen> {
...
                     return;
                   }
-                } catch (err) {
-                  print('HEY $err');
-                }
 
-                var content = "";
-                try {
-                  content = file.readAsStringSync();
+                  _addCertEntry(content, (err) {
```
Comment (JohnMaguire) [inline lib/screens/siteConfig/CertificateScreen.dart:228]: It seems potentially unnecessary to move this into the try`` - errors should invoke the callback here? I don't see any harm though.

File: lib/screens/siteConfig/CertificateScreen.dart [hunk 3]
```diff
@@ -219,34 +219,22 @@ class _CertificateScreenState extends State<CertificateScreen> {
...
+                    if (err != null) {
+                      Utils.popError(context, 'Error loading certificate file', err);
```
Comment (JohnMaguire) [inline lib/screens/siteConfig/CertificateScreen.dart:230]: Should this return? Previously this was the last line of code in this function - now the catch comes after it. I could see someone adding code following the catch, and not realizing that this doesn't return in the case of an error.

Comment (JohnMaguire) [reply lib/screens/siteConfig/CertificateScreen.dart:230]: Ignore me - this is occurring as part of a callback. The order confused me a bit.

**Review by JohnMaguire — APPROVED** [review]

File: lib/components/config/ConfigButtonItem.dart (hunk 1 above)
Comment (nbrownus) [reply lib/components/config/ConfigButtonItem.dart:24]: It's unreachable code I left around for whatever reason

**Pull request merged**